fjord/analytics/templates/analytics/analyzer/hb_surveys.html

```diff
@@ -1,74 +1,91 @@
 {% extends "analytics/analyzer/dashboard.html" %}
 {# Note: This is not l10n-ized since it's only available to analyzers for now. #}
 
-{% block content %}
-  <div class="col full">
-    <div class="block feedback">
-      <h2>Heartbeat v2 surveys</h2>
-      <p class="secret">
-        Do not tweet!
-      </p>
-      <p>
-        Note: If you need to create a new survey, ask Will. At some
-        point, we'll write a view that lets you add/edit
-        surveys. Sorry about the inconvenience.
-      </p>
+{% block content_middle %}
+<div class="block feedback">
```

Comment:
This makes sense as the page in which this block is nested will be properly indented if the contents of the block is indented by 2 spaces.